packages/compass/src/app/index.tsx

```diff
@@ -30,14 +30,17 @@ window.addEventListener('error', (event) => {
   );
 });
 
-window.addEventListener('unhandledrejection', (event) => {
-  event.preventDefault();
-  const error = ensureError(event.reason);
-  void ipcRenderer?.call('compass:error:fatal', {
-    message: error.message,
-    stack: error.stack,
-  });
-});
+window.addEventListener(
+  'unhandledrejection',
+  (event: PromiseRejectionEvent) => {
+    event.preventDefault();
+    const error = ensureError(event.reason);
+    void ipcRenderer?.call('compass:error:fatal', {
+      message: error.message,
+      stack: error.stack,
```

Comment:
Let's definitely distinguish between unhandled rejections and uncaught exceptions in the logs that we generate